Cos3rd.java

```diff
@@ -1,13 +1,32 @@
 class Cos3rd extends ConsoleProgram {
 
   /**
-  * Description
-  * @author:
+  * This program will receive input for two sides of a triangle and the contained angle, then to calculate the length of the third side
+  * @author: T. Martins
   */
   
   public void run() {
     
-    // start coding here
+    // declare variables
+    double dblSide1;
+    double dblSide2;
+    double dblAngle;
+    double dblSide3;
+
+    // get values of first side, second side, and contained angle
+    System.out.println("");
+    dblSide1 = readDouble("Input the first side of the triangle in centimetres: ");
+    dblSide2 = readDouble("Input the second side of the triangle in centimetres: ");
+    dblAngle = readDouble("Input the angle contained betweeen the two sides in degrees: ");
+    System.out.println("");
+
+    // convert degrees to radians, and calculate the third side
+    dblAngle = Math.toRadians(dblAngle);
+    dblSide3 = Math.sqrt(dblSide1 * dblSide1 + dblSide2 * dblSide2 - 2 * dblSide1 * dblSide2 * Math.cos(dblAngle));
```

Comment:
For another incomprehensible reason, I added * dblSide2 after dblSide1 * dblSide1 + dblSide2 *dblSide 2. Additionally, I could have used Math.pow(dblSide1, 2) * Math.power(dblSide2, 2) to simplify my code.